src/main/build/version.h

```diff
@@ -25,7 +25,7 @@
 #define FC_FIRMWARE_NAME            "EmuFlight"
 #define FC_VERSION_MAJOR            0  // increment when a major release is made (big new feature, etc)
 #define FC_VERSION_MINOR            2  // increment when a minor release is made (small new feature, change etc)
-#define FC_VERSION_PATCH_LEVEL      47 // increment when a bug is fixed
+#define FC_VERSION_PATCH_LEVEL      41 // increment when a bug is fixed
```

Comment:
nah now that should be 47 cause we cant get back in time can we

Reply:
well we are going to go to 0.3.0 soon. We just got over one of the hurdles we had with the gui so a full release is looking a lot sooner. I can change it back to 2.47 though

Reply:
so we should work on version `0.3.0` *now* and release that to the public when done. like lots of other projects release just fine working on one version until its done and then proceed